load-testing/audit-service/audit-presigned-url.js

```diff
@@ -20,44 +20,14 @@ export const options = {
 };
 
 export default function () {
-  // the url to post the logs to
   const auditURL = `https://${GEN3_HOST}/audit/log/presigned_url`;
```

Comment:
same comment here, i think the only difference between `audit-login.js` and `audit-presigned-url.js` is the category name in this URL